wordpress-mobile/WordPress-iOS
# Blogging Reminders - Fix ui issues

Fixes #NA

This PR fixes issues found [here](https://github.com/wordpress-mobile/WordPress-iOS/pull/17033#pullrequestreview-731138234)

This PR also introduces the simplified settings screen for older iPhones with lower resolution and for iPad landscape (see below). This simplified UI does not contain the calendar icon, the dismiss button (X) and the tip

Full UI | Simplified UI
-- | --
<img height="500"  src="https://user-images.githubusercontent.com/34376330/129982658-14b58b98-50e4-4970-842a-95da6f4cf557.png"> | <img height="500" src="https://user-images.githubusercontent.com/34376330/129982186-427764da-11df-4d6f-bdfc-12f4dc1ba734.png">


@osullivanchris please let me know your thoughts about iPad: we could also do simplified UI for both orientation if you think it's better, it's an easy fix.

**NOTE: this PR does not fix the transition animation on iPad, there will be a separate one for that issue**

To test:
- Build/run and go to My Site -> Site Settings and tap on "Blogging Reminders"
- If you had never set reminders for the current site, tap on "Get Started" on the intro screen. If you had, you can skip to the next step (you won't see the intro screen)
- On the day picker screen, try a variety of combinations between selecting/deselecting days, and going back and forth between the day picker and the time picker screen
- Make sure the UI looks good and the issues outlined [here](https://github.com/wordpress-mobile/WordPress-iOS/pull/17033#pullrequestreview-731138234) are not happening

## Regression Notes
1. Potential unintended areas of impact
This PR only changes UI elements in `BloggingRemindersFlowSettingsViewController`, any regression would be a UI issue in that same screen. No other part of the app is affected.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Test on a variety of simulators and devices

3. What automated tests I added (or what prevented me from doing so)
none
PR submission checklist:
